src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Main {
```

Comment:
Delete this class. It's ok if you need it for testing locally. Just do not commit it! )))